gapic/src/main/com/google/gapid/perfetto/canvas/Panel.java

```diff
@@ -26,6 +25,12 @@
     return Hover.NONE;
   }
 
+  @SuppressWarnings("unused")
+  /** Returns whether the screen should be redrawn. */
+  public default boolean onClick(Fonts.TextMeasurer m, double x, double y) {
+    return false;
+  }
+
```

Comment:
Please undo this. I see a lot of duplicated code due to this as we have to re-compute where the mouse is now that this is a panel-level method, rather than on the `Hover` object.

Reply:
Was trying to solve the de-selection problem in this way, but yeah it indeed included a lot of redundant code and the disadvantages outweighs the advantages. Have just reverted this part back, and found another way to solve the de-selection problem in RootPanel! Thanks Pascal!